src/ed25519/mod.rs

```diff
@@ -57,14 +46,7 @@ pub fn verify(
 ) -> TofnResult<bool> {
     let verifying_key = VerifyingKey::from_bytes(encoded_verifying_key).map_err(|_| TofnFatal)?;
 
-    let asn_signature = Asn1Signature::from_der(encoded_signature).map_err(|_| TofnFatal)?;
-    if asn_signature.signature_algorithm != ALGORITHM_ID {
-        return Err(TofnFatal);
-    }
-
-    // Using raw_bytes() here is safe since we do not have any unused bits.
-    let signature =
-        Signature::from_slice(asn_signature.signature.raw_bytes()).map_err(|_| TofnFatal)?;
+    let signature = Signature::from_slice(encoded_signature).map_err(|_| TofnFatal)?;
```

Comment:
feels weird to return fatal error parse verifying key and signature

Reply:
`TofnFatal` is the only public facing error `tofn` returns (I believe it was to avoid leaking any sensitive metadata from the threshold days). We can refine this in the future though now that gg20 is removed